metagraph/src/graph/annotated_dbg.hpp

```diff
@@ -74,6 +74,11 @@ class AnnotatedDBG : public AnnotatedSequenceGraph {
 
     const DeBruijnGraph& get_graph() const { return dbg_; }
 
+    // add k-mer counts to the annotation
+    void add_kmer_counts(std::string_view sequence,
+                         const std::vector<Label> &labels,
+                         std::vector<uint32_t>&& kmer_counts);
```

Comment:
maybe in the future... Right now we always have variable counts for k-mers.
For reads, the overhead is always negligible.
So this may be useful only for very long sequences, for which you don't want to add `vector<uint32_t>` overhead.